rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import commonjs from '@rollup/plugin-commonjs'
+import {nodeResolve} from '@rollup/plugin-node-resolve'
+import {babel} from '@rollup/plugin-babel'
+import {terser} from 'rollup-plugin-terser'
+
+const config = {
+  input: './lib/index.js',
+  output: {
+    file: './micromark.min.js',
+    format: 'umd',
+    name: 'Micromark',
+    plugins: [
+      // took from here https://github.com/browserify/tinyify/blob/default/index.js
+      terser({
+        // No need to mangle here, will do that at the end.
+        mangle: false,
+        output: {
+          // eslint-disable-next-line camelcase
+          ascii_only: true
+        }
+      }),
+      terser({
+        output: {
+          // eslint-disable-next-line camelcase
+          ascii_only: true
+        },
+        mangle: {
+          safari10: true
+        }
+      })
```

Comment:
Do you see differences for using `terser` twice? tinyify is running it on separate files, and on the whole, but this does the same twice. Maybe this isn’t noticable?

Reply:
13425 vs 13589. But yeah, single pass is more realistic for user build.

Reply:
ohhh, that’s significant! Weird! Well, let’s leave it in then